### PR TITLE
[vue] use default eslint for vue instead of tslint

### LINKED ISSUE #12764: Drop tslint in favor of eslint.
##### **Overview of the feature request**

Migrate to eslint.

##### **Motivation for or Use Case**

```
npm WARN deprecated tslint@6.1.2: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.
```

##### **Related issues or PR**

-   [ ] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by atomfrede**
Basically moves the changes done already in https://github.com/jhipster/jhipster-vuejs/pull/601 to the main generator.

Updates some 3rd party dependencies which have no breaking changes.

Fix #12764

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md do cument](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/vue/package.json.ejs [hunk 1]
```diff
@@ -27,14 +27,14 @@ limitations under the License.
   ],
   "dependencies": {
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
-    "@fortawesome/fontawesome-svg-core": "1.2.30",
-    "@fortawesome/free-solid-svg-icons": "5.14.0",
-    "@fortawesome/vue-fontawesome": "0.1.10",
-    "axios": "0.19.2",
-    "bootstrap": "4.5.0",
-    "bootstrap-vue": "2.14.0",
+    "@fortawesome/fontawesome-svg-core": "1.2.32",
```
Comment (DanielFran) [inline generators/client/templates/vue/package.json.ejs:30]: @atomfrede Can you migrate some of the dependencies to package.json file so the updates can be managed by dependabot automatically?

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Comment by DanielFran** [comment]
Closing/Reopening to launch only Vue tests and relaunch failing tests

**Comment by pascalgrimaud** [comment]
Just tested and it works well. Congrats @atomfrede ! Dont forget to claim your bounty at https://github.com/jhipster/jhipster-vuejs/issues/597

@DanielFran : your comment can be done in another PR I think. Let's merge this

**Comment by atomfrede** [comment]
Will have a look at the dependabot integration when upgrading e.g. the terser plugin and doing some preparations for vue3.

**Pull request merged**